python/packages/autogen-core/samples/adas/README.md

```diff
@@ -0,0 +1,515 @@
+- Documentation from official AutoGen website. Currently only parsing `.md` and `.ipynb` files from the [core-concepts](https://microsoft.github.io/autogen/dev/user-guide/core-user-guide/core-concepts/index.html) and [framework](https://microsoft.github.io/autogen/dev/user-guide/core-user-guide/framework/index.html) sections.
+
+### Meta-Agent does 5 iterations of LLM calls to create and edit code
+
+- The initial prompt to the meta-agent contains all the instructions (generate novel code with name and thought of the new system), output and formatting requirements, examples of how to use or not use the API, and historical context (archive) of previous Agent Systems.
+- 4 rounds of reflection:
+    - Round 1 to reflect on interestingness, implementation mistakes, and improvement
+    - Round 2 to revise based on the tips from the "Wrong Implementation" section in the original prompt
+    - Round 3 to revise based on the tips from the "Correct Implementation" section in the original prompt
+    - Round 4 to revise based on the tips from the official API documentation
+
+### Meta-Agent will try again fresh if it encounters (code compilation) errors when trying to execute
+
+An example of an exception is the following:
+```
+Error during evaluation:\nClosureAgent.register() takes 4 positional arguments but 5 positional arguments (and 1 keyword-only argument) were given\nCarefully consider where you went wrong in your latest implementation. Using insights from previous attempts, try to debug the current code to implement the same thought. Repeat your previous thought in 'thought', and put your thinking for debugging in 'debug_thought'", source='adas_agent')
+```
+The current behavior is that the meta-agent will start over the generation sequence (using the 5 steps of LLM calls).
+
+Note: The `adas.py` script can still get stuck even if the code of the agent system compiles. This can occur because the agent system itself hangs due to a message being published to a topic that is not used. See this [section](#the-code-that-the-meta-agent-does-not-compile) under the Troubleshooting section for details on how to address this issue.
+
+### Notable arguments to the script
+
+Please see the `adas.py` file for details of all available settings.
+
+- `data_filename`: the name of full path of the dataset location
+- `benchmark_specific_utils_file`: Benchmark-specific utility file to load the dataset and also evaluate the outputs. This file must contain the `load_dataset` and `compute_metrics` functions
+- `meta_agent_model_config`: JSON string of the `AzureOpenAIChatCompletionClient` settings for the Meta-Agent.
+- `base_agent_model_config`: JSON string of the `AzureOpenAIChatCompletionClient` settings for the Base Agent.
+- `n_generation`: number of generations of new agents that the meta-agent tries to discover
+- `expr_name`: name of the output file containing both the original/seed and newly generated agent systems, as well as their fitness scores.
```

Comment:
it was unclear where those files are stored. I could not find them.

Reply:
Added clarification.